autofs-5.0.7 - add changlog entry for coverity fixes

From: Ian Kent <raven@themaw.net>

A bunch of changes have been made based on a Covarity report.
Mostly I pust the changes into seperate patches so they can be kept simple.

Changes:
- fix fcntl return check.
- fix spawn_umount() return check in mount_bind.c:lookup_init().
- fix check mkdir_path() in mount_bind.c:mount_mount().
- fix incorrect var name in test.
- remove debug only code in alarm.c.
- fix inconsistent use of cache lock in handle_packet_missing_direct().
- fix several off by one errors.
- fix memory leak in get_dc_list().
- fix host_addr null reference in add_new_host().
- add null check in read_one().
- add pgrp check in do_spawn().
- fix inconsistent signed usage for __rpc_ping().
- add null check in extract_version().
- recheck valid map entry lookup return in do_readmap_mount().
- add null check in parse_server_string().
- add map entry null check in do_expire_direct().
- add mapent null check in lookup_nisplus.c:lookup_mount().
- fix potential null dereference in lookup_mount().
- fix leaked ldap percent hack allocation in lookup_one().
- fix incorrect value reference in parse_line().
- add debug alert for waitpid in check_nfs_mount_version().
- add initialization of bind_result in-do_sasl_bind().
- fix incorrect check in flag_is_owned().
- fix possible use after free in lookup_dir.c:lookup_init().
---
 CHANGELOG |    1 +
 1 file changed, 1 insertion(+)

diff --git a/CHANGELOG b/CHANGELOG
index 48e9806..25179b1 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -49,6 +49,7 @@
 - fix master map bogus keywork match.
 - fix fix map entry duplicate offset detection.
 - probe each nfs version in turn for singleton mounts.
+- add changlog entry for coverity fixes.
 
 25/07/2012 autofs-5.0.7
 =======================
